src/views/Notifications/Home/_components/DailyDigest.tsx

```diff
@@ -112,7 +107,7 @@ const DailyDigest = ({
                       })}
                     </li>
                   )}
-                {planSections &&
+                {planSections?.readyForClearance &&
                   planSections.readyForClearance.length > 0 &&
```

Comment:
Just a note for these, you can chain one further:

`planSections?.readyForClearance?.length > 0 `